apps/web/src/app/[locale]/[...path]/page.tsx

```diff
@@ -96,6 +99,14 @@ async function Page({ params: { path } }: Props) {
       return <EventsPage />;
     }
 
+    if (page.specialPageType === "weekly-newsletter") {
+      return <WeeklyNewsletterPage />;
+    }
+
+    if (page.specialPageType === "weekly-newsletters-list") {
+      return <WeeklyNewslettersListPage />;
+    }
+
```

Comment:
Is there a reason we have nested typing, i.e. `page.type` is now special and that special page has more than one type itself. Why not just have these in `page.type` instead? Or at least it seems like this feels like unnecesarily nested logic, and with the other pattern we can give each special page their own custom fields.

Reply:
I just thought that there might be shared fields between all custom pages, like if we had route params there. I'll think about it

Reply:
I guess it makes sense to just be under `page.type`, couldn't come up with any reason we would have shared fields between all special-pages and we can do it then